src/Components/Forms/OTPForm.tsx

```diff
@@ -1,12 +1,43 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import OTPInput, { ResendOTP } from "otp-input-react";
 import otp_clipArt from "../../assets/Images/OTP_clipArt.png";
-import { useState } from "react";
+import { useContext, useEffect, useState } from "react";
+import UserContext from "../../contexts/UserContext";
+import axios from "axios";
+import { baseURL } from "../../const/const";
+import { useNavigate } from "react-router";
 const OTPForm = () => {
+  axios.defaults.withCredentials = true;
   const [OTP, setOTP] = useState("");
   const [error, setError] = useState("");
-
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const phone = "9876543210";
+  const { user } = useContext(UserContext);
+  const navigate = useNavigate();
+  const phone = user.phone;
+  useEffect(() => {
+    axios.post(baseURL + "/user/sendOTP", { phone_number: phone });
```

Comment:
what is the reason for this post request?

Reply:
noted. I'll Fix that